app/assets/javascripts/short-form/components/preferenceCheckbox.js.coffee

```diff
@@ -9,16 +9,20 @@ angular.module('dahlia.components')
     translatedDescription: '@'
     customDescription: '@'
     shortDescription: '@'
+    readMoreUrl: '@'
     onChange: '&'
   templateUrl: 'short-form/components/preference-checkbox.html'
 
   controller: ['ListingDataService', 'ListingPreferenceService', (ListingDataService, ListingPreferenceService) ->
     ctrl = @
 
-    @showDescription = false
     listingPreference = ListingPreferenceService.getPreference(@preference, ListingDataService.listing)
     if listingPreference
       @moreInfoLink = listingPreference.readMoreUrl
+    else
+      # If you can't find the moreInfo link, for example, with a custom preference
+      # like 588 Mission, look for a passed readMoreUrl value
```

Comment:
I tried to add 588 mission to comments in wonky places that we had to update but could be a bit more consistent